package.json

```diff
@@ -1,24 +1,25 @@
 {
```

Comment:
Confirmed, it was npm init what added that 'git+' prefix and the empty dependencies object by default. And for some reason it also changes the directories and removes two of them. What's odd is that it leaves the rest intact. The documentation of [npm-init](https://docs.npmjs.com/cli/init) is not too good.